simd.dd

```diff
@@ -60,6 +60,9 @@ import core.simd;
 	the compiler is targetting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
 	The compiler does not emit runtime checks; those must be done by the programmer.
+    For instance, the Windows version of DMD only implements support for SIMD when
```

Comment:
I don't know how true it is on other machines. I was just providing an example for what I was working on.